### PR TITLE
[chore] - Sentry detector update

### PR DESCRIPTION
**Pull request by ahrav**
### Description:
Add test case for account deactivated and when the API returns an empty result array. `[]`

### Checklist:
* [ ] Tests passing (`make test-community`)?
* [ ] Lint passing (`make lint` this requires [golangci-lint](https://golangci-lint.run/usage/install/#local-installation))?

### CODE REVIEW

File: pkg/detectors/sentrytoken/sentrytoken_test.go [hunk 1]
```diff
@@ -174,6 +209,9 @@ const responseBody403 = `
   }
 ]
 `
+	reponseAccountDeactivated = `{"datail": "Authentication credentials were not provided"}`
```
Comment (0x1) [inline pkg/detectors/sentrytoken/sentrytoken_test.go:212]: should this be `detail`?

**Review by 0x1 — APPROVED** [review]
lgtm other than clarification on `datail`

File: pkg/detectors/sentrytoken/sentrytoken_test.go (hunk 1 above)
Comment (ahrav) [reply pkg/detectors/sentrytoken/sentrytoken_test.go]: ah yea, it shouldn't affect the test since we are expecting a completely different structure. Nonetheless, fixed. TY.
